src/Utility/Functions.C

```diff
@@ -434,17 +438,15 @@ ScalarFunc* utl::parseTimeFunc (const char* func, const std::string& type)
 }
 
 
-RealFunc* utl::parseRealFunc (const std::string& func, const std::string& type, bool print)
+RealFunc* utl::parseRealFunc (const std::string& func, const std::string& type)
 {
   if (func.empty()) return nullptr;
 
-  if (print)
-    IFEM::cout <<": ";
+  IFEM::cout <<": ";
   Real p = Real(0);
   if (type == "expression")
   {
-    if (print)
```

Comment:
just checking. this was used to suppress output during ic parsing, you have reorganized so the output doesn't mess up ?